src/chart/rendering/colorbuilder.h

```diff
@@ -26,7 +26,7 @@ struct ColorBuilder
 	    const Gen::ColorBase &colorBase) const;
 
 private:
-	LighnessRange lighnessRange{0.4, -0.4};
+	LighnessRange lighnessRange = LighnessRange::Raw(-0.4, 0.4);
```

Comment:
@simzer Is this the expected working?

Reply:
If I swap the arguments, all test cases run without any difference.

Reply:
So this parameter (this construction) is unused. The only constructor initializes this parameter through argument, not by default.

Reply:
![kép](https://github.com/vizzuhq/vizzu-lib/assets/6457941/f2caccee-2cb3-4ddc-aee0-d542e043dd41)
The default argument is (-0.4, 0.4).